data/assets/css/components/_event-summary.scss

```diff
@@ -67,6 +71,9 @@
     height: 2px;
     color: black;
     background-color: black;
-    margin: $space-large auto $space-larger auto;
+    margin: $space-large/2 auto $space-larger/2 auto;
```

Comment:
same here, try with $space-medium in both or medium - large instead of the divisions, thx!